ios/browser/api/brave_rewards/brave_rewards_api.mm

```diff
@@ -1197,17 +1040,14 @@ - (void)startNotificationTimers {
     self.notificationStartupTimer =
         [NSTimer scheduledTimerWithTimeInterval:30
                                          target:self
-                                       selector:@selector
-                                       (checkForNotificationsAndFetchGrants)
+                                       selector:@selector(checkForNotifications)
                                        userInfo:nil
                                         repeats:NO];
   });
 }
 
-- (void)checkForNotificationsAndFetchGrants {
+- (void)checkForNotifications {
```

Comment:
This entire method can just be deleted (along with the NSTimer created that periodically calls it) now that its not going to do anything in here anymore. The `lastNotificationCheckDate` property can also be removed

Reply:
Done.